.github/workflows/step_lifecycle.yaml

```
@@ -0,0 +1,104 @@
+on:
+  workflow_call:
+    inputs:
+      next_step:
+        required: true
+        type: string
+jobs:
+  load_vars:
+    name: Load Variables
+    uses: ./.github/workflows/vars.yml
```

actionlint (1.7.7)

could not read reusable workflow file for "./.github/workflows/vars.yml": open /home/jailuser/git/.github/workflows/vars.yml: no such file or directory

(workflow-call)